dolphinscheduler-python/pydolphinscheduler/src/pydolphinscheduler/resources_plugin/__init__.py

```diff
@@ -0,0 +1,37 @@
+import importlib
+import importlib.util
+
+from pathlib import Path
+from typing import Any, Generator
+
+from pydolphinscheduler.exceptions import PyDSConfException
+
+path_resources_plugin = Path(__file__).parent
+
+
+# [start resource_plugin_definition]
+class ResourcePlugin:
+    def __init__(self, type: str, prefix: str):
+        self.type = type
+        self.prefix = prefix
+
+    def get_all_modules(self) -> Generator[Path, Any, None]:
```

Comment:
can we directly use `Path`here?
```suggestion
    def get_all_modules(self) -> Generator[Path]:
```